app/v1/models/request/disclosures/AmendDisclosuresRequestBody.scala

```diff
@@ -16,11 +16,12 @@
 
 package v1.models.request.disclosures
 
-import play.api.libs.json.{Json, Reads}
+import play.api.libs.json.{Json, OWrites, Reads}
 
-case class DisclosuresRequestBody(taxAvoidance: Option[Seq[TaxAvoidance]])
-
-object DisclosuresRequestBody {
-  implicit val reads: Reads[DisclosuresRequestBody] = Json.reads[DisclosuresRequestBody]
+case class AmendDisclosuresRequestBody(taxAvoidance: Option[Seq[AmendTaxAvoidance]])
 
+object AmendDisclosuresRequestBody {
+  val empty = AmendDisclosuresRequestBody(None)
+  implicit val reads: Reads[AmendDisclosuresRequestBody] = Json.reads[AmendDisclosuresRequestBody]
```

Comment:
Use JsPath for the reads and writes as mapping change

Reply:
Changed for writes. Looking at the other service we never use path for single field models on reads, so I've left reads the same

Reply:
Which service are you looking at?

Reply:
which service are you looking at?